telethon/events/common.py

```diff
@@ -145,9 +147,9 @@ def __init__(self, chat_peer=None, msg_id=None, broadcast=None):
         self._entities = {}
         self._client = None
         self._message_id = msg_id
-        self.original_update = None
+        self.original_update: typing.Optional[types.TypeUpdate] = None
```

Comment:
The library currently supports Python 3.5, and this won't work there. Only hints in parameters.

Reply:
Fine, but I think it will limit the ability of type-hinting in situations like this (And Python 3.5 is EOL btw).

Reply:
I know it's EOL, but no longer supporting a version is a breaking change for those using that old version. I believe IDEs like IntelliJ detect certain comments as type hints if you prefer.